components/brave_adblock_ui/adblock_internals/components/app.tsx

```diff
@@ -15,37 +15,91 @@ class AppState {
   memory: { [key: string]: string } = {}
 }
 
+const EngineInfo = (props: { engine: EngineDebugInfo; caption: string }) => {
+  return (
+    <div>
+      <h2>{props.caption}</h2>
+      <div>
+        Flatbuffer size:{' '}
+        {(props.engine.flatbuffer_size / 1024 / 1024).toFixed(2)} MB
+      </div>
+      <div>Compiled regexes: {props.engine.compiled_regex_count}</div>
+    </div>
+  )
+}
+
 export class App extends React.Component<{}, AppState> {
```

Comment:
optional: while we're here we could conver this to a functional component as Class Components are the old way of doing things

Reply:
Done, thanks!